<launch>
  <arg name="model_dir" default="/media/shintarosakoda/5EA85517A854EF51/diffusion_planner_training_result/train_result/20250610-110759_with_psim_data/"/>

  <node pkg="diffusion_planner_ros" exec="diffusion_planner_node" name="diffusion_planner_node_python" output="both">
    <param name="vector_map_path" value="$(var map_path)/lanelet2_map.osm"/>
    <param name="config_json_path" value="$(var model_dir)/args.json"/>
    <param name="ckpt_path" value="$(var model_dir)/latest.pth"/>
    <param name="onnx_path" value="$(var model_dir)/model.onnx"/>
    <param name="backend" value="PYTORCH"/>
    <param name="batch_size" value="1"/>
    <param name="use_sim_time" value="true"/>
  </node>
</launch>
